controllers/articles.js

```diff
@@ -1,9 +1,24 @@
 
 var mongoose  = require('mongoose'),
-  moment    = require('moment'),
-  Article   = mongoose.model('Article');
+  moment      = require('moment'),
+  Article     = mongoose.model('Article'),
+  passport    = require('passport'),
+  express     = require('express'),
+  auth        = require('./../middlewares/auth');
 
-exports.view = function (req, res) {
+module.exports = (function () {
```

Comment:
You don't understand what I mean. I know that the top-level scope is local to the file in node, it can't be a source of conflict. 

However I avoid statements in the global scope *of the file* as much as I can because it becomes messy in a very short time. Your code in the scope of the file act as "the constructor" of the module. It becomes difficult to track what is made in this "constructor" when your file becomes bigger because the initialization statements can be separated by huge functions declarations.

I know meticulous developers wont do that but the world is not full of meticulous developers. The IIFE force the developer to declare all routes of a file in a single place, I think this is a good thing, and there is a reasoning behind.